### PR TITLE
chore: Update reqwest to 0.9

### PR DESCRIPTION
**Pull request by igor-raits**
Signed-off-by: Igor Gnatenko <i.gnatenko.brain@gmail.com>

### CODE REVIEW

File: src/request.rs [hunk 1]
```diff
@@ -38,20 +38,20 @@ impl Request {
...
         }
 
         if let Some(content_type) = self.content_type.clone() {
-            match content_type {
-                ContentType::Json => builder.header(header::ContentType::json()),
+            builder = match content_type {
+                ContentType::Json => builder.header(header::CONTENT_TYPE, "application/json"),
                 ContentType::FormUrlEncoded => {
-                    builder.header(header::ContentType::form_url_encoded())
+                    builder.header(header::CONTENT_TYPE, "application/x-www-form-urlencoded")
```
Comment (iddm) [inline src/request.rs]: This could be a static const string. Could you please move it to the top so?

File: src/request.rs [hunk 2]
```diff
@@ -38,20 +38,20 @@ impl Request {
...
         if let Some(agent) = self.user_agent.clone() {
-            builder.header(header::UserAgent::new(agent.0));
+            builder = builder.header(header::USER_AGENT, agent.0);
         }
 
         if let Some(content_type) = self.content_type.clone() {
-            match content_type {
-                ContentType::Json => builder.header(header::ContentType::json()),
+            builder = match content_type {
+                ContentType::Json => builder.header(header::CONTENT_TYPE, "application/json"),
```
Comment (iddm) [inline src/request.rs]: This could be a static const string. Could you please move it to the top so?

**Review by iddm — APPROVED** [review]
A good job has been done, thank you for participating and updating to the newest version of reqwest crate. However, the code you have touched could also be improved a bit by moving constants to the top. I'd like your doing that. Also, why did you put "chore: " into the commit title? :)

**Comment by igor-raits** [comment]
> Also, why did you put "chore: " into the commit title? :)

a routine task, especially a household one.

---

Suggestions are implemented.

**Review by iddm — APPROVED** [review]
Amazing!

**Comment by iddm** [comment]
Uploaded new version with this PR: https://crates.io/crates/urlshortener

**Pull request merged**